lib/mpool/lib/mblock_file.c

```diff
@@ -896,17 +899,50 @@ mblock_file_meta_validate(const struct mblock_file *mbfp, uint64_t *mbidv, int m
     return err;
 }
 
+static merr_t
+mblock_alen_get(struct mblock_file *mbfp, uint64_t mbid, uint32_t *alen)
+{
+    off_t start, cur, end;
+    size_t mblocksz;
+
+    INVARIANT(mbfp && alen);
+
+    mblocksz = mbfp->mblocksz;
+    start = block_off(mbid, mblocksz);
+    end = start + mblocksz - 1;
+
+    cur = lseek(mbfp->fd, start, SEEK_SET);
+    assert(cur == start);
```

Comment:
This will assert if `lseek` returns an error. That doesn't seem right.

Reply:
Good point, will remove. Thnx!